mp/src/game/server/momentum/mom_triggers.h

```diff
@@ -688,3 +688,52 @@ class CNoGrenadesZone : public CBaseTrigger
 
     bool m_bAirborneOnly;
 };
+
+// CTriggerMomentumCatapult
+class CTriggerMomentumCatapult : public CBaseMomentumTrigger
+{
+  public:
+    DECLARE_CLASS(CTriggerMomentumCatapult, CBaseMomentumTrigger);
+    DECLARE_DATADESC();
+
+    CTriggerMomentumCatapult();
+
+  public:
+    void OnStartTouch(CBaseEntity *) OVERRIDE;
+    void Spawn() OVERRIDE;
```

Comment:
Use `override` instead of `OVERRIDE`
```suggestion
    void Spawn() override;
```